TITLE:
Multiple issues when project name includes special symbolds s.a. ":".

LABEL:
bug

STATE:
closed

BODY:
* Weights and Biases version: 0.8.7
* Python version: 3.6.9
* Operating System: Ubuntu 18.04

### Problem
During my `wandb.init(project=<project_name>,...)`-call with a **project name that includes special characters s.a. ":"**, I get a `wandb.apis.CommError: <Response [400]>` error or `requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://api.wandb.ai/graphql` without any information on the problem source. In addition, this results in me not being able to delete these projects from my online wandb page. If I try to do that, I get an `invalid project id`-error, and the project remains.

### Temporary Fix
From my side, the fix was very simple: I simply did not use any special characters. 

However, I now have multiple dead projects that spam my Project page which I assume can't be removed due to the special character issue.




